calibration/CTDMOG/CGINS-CTDMOG-13604__20170331.csv

```diff
@@ -22,4 +22,4 @@ serial,name,value,notes
 37-13604,CC_ptempa0,1.738165e+002,
 37-13604,CC_ptempa1,-8.077719e-002,
 37-13604,CC_ptempa2,2.353830e-006,
-37-13604,CC_wbotc,4.6450e-007,
+37-13604,CC_wbotc,4.644981e-07,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT